src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -46,6 +46,7 @@ public partial class MarkStep : IStep {
 		protected Queue<AttributeProviderPair> _assemblyLevelAttributes;
 		protected Queue<AttributeProviderPair> _lateMarkedAttributes;
 		protected List<TypeDefinition> _typesWithInterfaces;
+		protected List<MethodBody> _pendingBodies;
```

Comment:
Please rename it to a better name, e.g. `_unreachableBodies`

Reply:
Done